hardhat/README.md

```diff
@@ -237,6 +239,20 @@ To solve this problem:
    npx hardhat --network optimistic test
    ```
 
+## Deploying to a Real Network
+
+To deploy to a real network (Optimistic Ethereum or Optimistic Kovan),
+edit `hardhat.config.js`'s `modules.export.networks` to add a definition
+similar to this one:
+
+```javascript
+    optimisticKovan: {
```

Comment:
Should we standardise the network names we use in examples? So far I've been trying to stick to `optimism`, `optimism-kovan` and `optimism-mainnet` for the networks. The other related terms are: "Optimistic rollups", "Optimistic Ethereum". Not sure which is better or whether they need to be synced. What do you think @qbzzt ?

Reply:
I changed to `optimistic` because somebody (I think it was Kevin, but I'll have to dig that up) told me we're trying to get people to distinguish between **Optimism** the company and **Optimistic Ethereum** the software. So I'd rather change this to `optimistic-kovan`. OK?

Reply:
Okay fine by me, we need a PR that makes that consistent everywhere